### DISCUSSION TITLE
切换深色模式时，有些组件的颜色切换transition过渡时间不太一致

### DISCUSSION THREAD

**Discussion by nanvon** [vueComponent/ant-design-vue#6380] [Q&A]
我对官网文档进行了录屏，慢速逐帧播放 GIF，可以发现菜单的颜色过渡慢于其他组件，这种该如何解决呢？
![深色模式切换](https://user-images.githubusercontent.com/36064282/227143739-1417c829-5e3c-4584-ad4b-90159324a167.gif)

**Comment by songpeng154** [comment]
切换之前修改这些Dom的transitiion的过渡时间，修改成一样的
